app/code/Magento/Sales/Model/Order/Creditmemo.php

```diff
@@ -425,23 +425,6 @@ public function canCancel()
     public function canVoid()
     {
         return false;
-        $canVoid = false;
-        if ($this->getState() == self::STATE_REFUNDED) {
```

Comment:
It would be really nice to analyze from git history when this was hardcoded to `return false`, maybe a part of functionality is broken now.

Reply:
@orlangur 
![image](https://user-images.githubusercontent.com/5318512/37466916-94aef26e-2867-11e8-94f0-ca01b9b544a1.png)
As I can see this was from the very beginning.